queries/cairo/highlights.scm

```diff
@@ -0,0 +1,340 @@
+; Preproc
+
+[
+  "%builtins"
+  "%lang"
+] @preproc
+
+; Includes
+
+(import_statement [ "from" "import" ] @include module_name: (dotted_name (identifier) @namespace . ))
+
+[
+  "as"
+  "use"
+  "mod"
+] @include
+
+; Variables
+
+(identifier) @variable
+
+; Namespaces
+
+(namespace_definition (identifier) @namespace)
+
+(mod_item
+  name: (identifier) @namespace)
+
+(use_list (self) @namespace)
+
+(scoped_use_list (self) @namespace)
+
+(scoped_identifier (self) @namespace)
```

Comment:
Shouldn't `(self)` be `@variable.builtin`?

Reply:
I just took that from the Rust highlights since, functionally, they're the same in Cairo, but I would agree with you